### ISSUE TITLE
Nail down the options for creating a scenario

### ISSUE THREAD

**Issue by lgoldstien** [thejsninja/syrup#7]

**Comment by lgoldstien** [comment]
After some thought, I'm moving a an object definition for declaring scenarios:

```javascript
syrup.scenario({
        name: 'array',
        notes: 'Test that standard array methods work',
        entrypoint: `${__dirname}/test-array`,
        dependsOn: [],
        worker: 'Console'
    });
```

**Comment by lgoldstien** [comment]
fixed in 4ca7fc724a320c6b37b92beaf54e91a3f8981452
